server/views/pages/activities/manage-allocations/allocateMultiplePeople/confirmation.njk

```diff
@@ -0,0 +1,32 @@
+{% extends "layout.njk" %}
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+
+{% set pageTitle = applicationName + " - Allocate to an activity - Confirmation" %}
+{% set pageId = 'multiple-allocation-confirmation-page' %}
+{% set activityId = session.allocateJourney.activity.activityId %}
+{% set activityName = session.allocateJourney.activity.name %}
+
+{% if session.allocateJourney.inmates | length > 1 %}
+    {% set panelText = session.allocateJourney.inmates | length + ' people are now allocated to ' + activityName %}
+{% else %}
+    {% set panelText = session.allocateJourney.inmates[0] | formatName(NameFormatStyle.firstLast, false) + ' is now allocated to ' + activityName %}
+{% endif %}
+
+{% block content %}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+                {{ govukPanel({
+                    titleText: "Allocations complete",
+                    text: panelText
+                }) }}
+                <h2 class="govuk-heading-m">What you can do next</h2>
+                <p class='govuk-body'>You can:</p>
+                <ul class='govuk-list govuk-list--bullet'>
+                    <li><a href="/activities/allocation-dashboard/{{ activityId }}" class='govuk-link' data-qa="activity-page-link">review the allocations for {{ activityName }}</a> and continue allocating</li>
+                    <li><a href="/activities/allocation-dashboard" class='govuk-link' data-qa="allocations-dash-link">find other activities with vacancies</a></li>
+                    {# TODO: haven't been told where this link is meant to go yet #}
+                    {# <li><a href="#" class='govuk-link'>find someone to manage their allocations</a></li> #}
```

Comment:
There's an extra link here in the designs, but I haven't been told where it should go yet (I've asked, but Ethan is unsure). Might need to wait until Stuart is back, so I have commented the link out for now.